packages/arb-provider-ethers/src/lib/provider.ts

```diff
@@ -149,47 +154,6 @@ export class ArbProvider extends ethers.providers.BaseProvider {
     )
   }
 
-  // public async getValidatorAddresses(): Promise<string[]> {
```

Comment:
Fine to delete all of this, but try to keep feature PRs isolated to just the work of adding that feature rather than including other unrelated changes.